static/app/views/explore/savedQueries/savedQueriesLandingContent.tsx

```diff
@@ -1,14 +1,84 @@
+import {useState} from 'react';
+import styled from '@emotion/styled';
+
+import {CompactSelect} from 'sentry/components/core/compactSelect';
+import SearchBar from 'sentry/components/searchBar';
+import {IconSort} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {SortOption} from 'sentry/views/explore/hooks/useGetSavedQueries';
 
 import {SavedQueriesTable} from './savedQueriesTable';
 
+type Option = {label: string; value: SortOption};
+
 export function SavedQueriesLandingContent() {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [sort, setSort] = useState<SortOption>('recentlyViewed');
+  const sortOptions: Option[] = [
+    {value: 'recentlyViewed', label: t('Recently Viewed')},
+    {value: 'name', label: t('Name')},
+    {value: 'dateAdded', label: t('Date Added')},
+    {value: 'dateUpdated', label: t('Date Updated')},
+    {value: 'mostPopular', label: t('Most Popular')},
+  ];
   return (
     <div>
-      <h4>{t('Owned by Me')}</h4>
-      <SavedQueriesTable mode="owned" />
-      <h4>{t('Shared with Me')}</h4>
-      <SavedQueriesTable mode="shared" perPage={8} />
+      <FilterContainer>
+        <SearchBarContainer>
+          <SearchBar
+            onChange={setSearchQuery}
+            value={searchQuery}
+            placeholder={t('Search for a query')}
+          />
+        </SearchBarContainer>
+        <CompactSelect
+          triggerProps={{
+            icon: <IconSort />,
+            size: 'md',
+          }}
+          triggerLabel={sortOptions.find(option => option.value === sort)?.label}
+          options={sortOptions}
+          value={sort}
+          onChange={option => setSort(option.value)}
+        />
+      </FilterContainer>
+      {searchQuery.length > 0 ? (
+        <span>
```

Comment:
Any reason why these need to be inside a span? Technically spans should only have certain kinds of children. It doesn't seem to typically result in UI bugs but just worth a callout

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span#technical_summary (see Permitted content)

Reply:
removed the unnecessary spans!